_studio/shared/include/mfx_utils.h

```diff
@@ -37,33 +37,13 @@
 #include <va/va.h>
 #endif
 
-#ifndef MFX_DEBUG_TRACE
-#define MFX_STS_TRACE(sts) sts
-#else
-template <typename T>
-static inline T mfx_print_err(T sts, const char *file, int line, const char *func)
-{
-    if (sts)
-    {
-        printf("%s: %d: %s: Error = %d\n", file, line, func, sts);
-    }
-    return sts;
-}
-#define MFX_STS_TRACE(sts) mfx_print_err(sts, __FILE__, __LINE__, __FUNCTION__)
-#endif
+#include "mfx_utils_defs.h"
```

Comment:
I don't get why you move part (and only part) of the definitions into other file. What is the rational under that? Why you can't use mfx_utils.h as is?

And why you move  most of MFX_CHECK defines, but MFX_CHECK_NO_RET is being left here? align2_value also remains here?

There should be some logic behind, but I don't get it.

Reply:
There are many "include" in mfx_utils.h (such as umc). It is not needed in encTools.